squire_sdk/src/client/network.rs

```diff
@@ -61,7 +66,11 @@ pub enum NetworkCommand {
 
 #[async_trait]
 impl ActorState for NetworkState {
+    type Permanence = Transient;
```

Comment:
`NetworkState` attaches to a `SquireClient` which is created at the top level of `main`. I guess it should be `Permanent`.

(I initially thought it was a per-request sort of thing because I misunderstood what `url` was)

Reply:
Yes, there is no reason for this to fail.